website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -744,6 +752,8 @@ and consider if they're appropriate for your deployment.
       "sample/annotation2": "bar"
     ```
 
+    - `ipFamilyPolicy` (`string: ""`) - The IP family and IP families options are to set the behaviour in a dual-stack environment. Omitting these values will let the service fall back to whatever the CNI dictates the defaults should be. These are only supported for kubernetes versions >=1.23. The service's supported IP family policy, can be either: SingleStack: Single-stack service. The control plane allocates a cluster IP for the Service, using the first configured service cluster IP range. PreferDualStack: Allocates IPv4 and IPv6 cluster IPs for the Service. RequireDualStack: Allocates Service .spec.ClusterIPs from both IPv4 and IPv6 address ranges.
+
   - `serviceAccount` - Values that configure the Kubernetes service account created for Vault.
 
     - `create` (`boolean: true`): If set to true, creates a service account used by Vault.
```

Comment:
Looks like [server.serviceAccount.createSecret](https://github.com/hashicorp/vault-helm/blob/f72df27d56698ea56b7392a7a83cfbcb0411006d/values.yaml#L949-L954) should be included somewhere around here.